### PR TITLE
Documentation update for sensor.hp_ilo

### PR DESCRIPTION
**Pull request by Juggels**
**Description:**
Documentation update for the HP ILO sensor component. Some major redesign was done on this component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7534

### CODE REVIEW

File: source/_components/sensor.hp_ilo.markdown [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: page
 title: "HP ILO"
 description: "How to integrate HP ILO (Integrated Lights-Out) sensors within Home Assistant."
-date: 2016-08-15 19:00
+date: 2017-05-10 19:00
```
Comment (Landrash) [inline source/_components/sensor.hp_ilo.markdown]: Leave the date as it was since the creation date of this document hasn't changed.

Comment (Juggels) [inline source/_components/sensor.hp_ilo.markdown]: Allright, will change that. Thought that it was the date of last modification.

**Comment by Juggels** [comment]
parent PR has been merged

**Comment by Landrash** [comment]
Thank you 🎂

**Comment by Juggels** [comment]
Hi @Landrash, is there something wrong in the rendering of this page? The values for 'value_template' in the example section don't seem to be generated in the end result.

**Pull request merged**